nav2_smac_planner/src/smac_planner_hybrid.cpp

```diff
@@ -386,6 +391,56 @@ nav_msgs::msg::Path SmacPlannerHybrid::createPlan(
     plan.poses.push_back(pose);
   }
 
+
+  // Publish expansions for debug
+  if (_viz_expansions) {
+    auto marker_array = std::make_unique<visualization_msgs::msg::MarkerArray>();
+    visualization_msgs::msg::Marker m;
```

Comment:
create a utils.hpp function called `createMarker(Frame, Time)` which creates `m` and returns it so that this isn't inline in the codebase.

Reply:
Yes, I actually plan to consolidate these functions used for visualizing planning results into one file.